### PR TITLE
Update charter to reflect current status

### PR DESCRIPTION
**Pull request by mabelzhang**
# Modify Governance Model

As Arjo and I discussed

Updating WG charter to reflect the current status:
- Move governance to the top so the key WG links are easily seen
- Update participant roles to be more lenient, since we don’t have formal membership
- Replace “subprojects” with “reference projects,” “Repositories” with “Proposals” since they’re all proposals. Remove rules about subprojects, since we do not anticipate the time and labor to maintain code in the near future. We can add the structure back as the need and availability arise.

Terminology:
- Replace “unmanned” with ungendered term “uncrewed.” I didn’t change to autonomous because there is a difference between uncrewed (e.g. remotely operated vehicles) and autonomous. All autonomous vehicles are uncrewed, but not all uncrewed vehicles are autonomous. So it is a more inclusive term.
- Replace some instances of “maritime” with “marine.” It is a more inclusive term. All maritime (seafaring, traveling, shipping) vehicles may be considered marine, but not all marine vehicles (e.g. used in underwater science) are maritime. I didn’t touch the name of the WG.

### CODE REVIEW

**Review by arjo129 — APPROVED** [review]

**Comment by mabelzhang** [comment]
@matosinho Could you take a look please? We need 2 approvals per the rules.

**Comment by mabelzhang** [comment]
[2a99ba0](https://github.com/ros-maritime/community/pull/10/commits/2a99ba0fe3c4130e8498f73c303f1faac90c3f29) Added the sailboat simulator from https://github.com/ros-maritime/issue_tracking/issues/13 that we moved to Done column in the kanban and said we'd add to the list.
Added a few other simulators from OSRF while at it.

**Comment by mabelzhang** [comment]
@arjo129 @matosinho Do you guys think requiring 2 approvals is still realistic for this charter? If we cannot get 2 approvals in a timely manner going forward, I suggest just changing that in this PR to 1 approval. There's nothing THAT critically important in this charter, now that we are removing the very technical parts like buildfarm requirements.

File: README.md
Comment (srmainwaring) [inline README.md]: Update to current version for Gazebo Garden: [Wave Sim](https://github.com/srmainwaring/asv_wave_sim)

File: README.md
Comment (srmainwaring) [inline README.md]: Remove the Docker reference for sail_sim_docker, it's for a previous version that runs on Gazebo Classic.

**Review by srmainwaring — COMMENTED** [review]
Looks good @mabelzhang - thanks for the update. Provided a couple of updates for the wave sim project. Happy to approve if you need another tick to merge.

File: README.md
Comment (mabelzhang) [reply README.md]: [3bc9e5b](https://github.com/ros-maritime/community/pull/10/commits/3bc9e5b27d9ee9abe3af69006275fcdde0aa4577)

File: README.md
Comment (mabelzhang) [reply README.md]: [3bc9e5b](https://github.com/ros-maritime/community/pull/10/commits/3bc9e5b27d9ee9abe3af69006275fcdde0aa4577)

**Comment by mabelzhang** [comment]
Thanks @srmainwaring! Updated.
A tick helps but I guess the charter says requires 2 approvals from Approvers / Maintainers. Shrugs. Wouldn't hurt to have another tick though. I'm thinking whether the 2 approvals are worth it since it seems to delay simple changes.

**Comment by arjo129** [comment]
I do feel expecting two approvers for this is a bit unreasonable, given that most of us do this part time.

**Comment by mabelzhang** [comment]
Okay thanks. I've updated to 1 approval required in e02a10a

**Comment by mabelzhang** [comment]
I'm gonna merge this as discussed above, so that it is done before the next monthly meeting.
If there are disagreements, feel free to comment.

**Pull request merged**
